updater.py

```diff
@@ -90,6 +93,29 @@ def do_update(args):
     current_link.symlink_to(release_dir, target_is_directory=True)
     print('Deployed', release_name)
 
+    do_cleanup(releases_dir, release_name)
+
+
+def do_cleanup(releases_dir, latest_release):
```

Comment:
this is just to keep the size of the project down locally right?

Reply:
The webserver was stuffed full of old versions at last check. The idea is that it only keeps the 5 most recent versions.

Reply:
So it's for production usage, rather than locally really (I don't use the deployer locally aside from in test)